### PR TITLE
Aavechan/weETH risk parameters update

### PR DESCRIPTION
**Pull request by Rozengarden**
This PR contain the code of the AIP to update the risks parameters of the weETH asset on mainnet AaveV3.

link: https://governance.aave.com/t/arfc-updating-weeth-risk-parameters/17402

### CODE REVIEW

File: src/20240426_AaveV3Ethereum_UpdatingWeETHRiskParameters/UpdatingWeETHRiskParameters.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+## Specification
+
+The following changes will be made to the **weETH** asset on Aave EthereumV3:
+
+| Parameters | Current | Recommended |
+| Supply Cap | 48,000 | 84,000 |
+| Borrows Caps | 3,200 | 29,500 |
+| UOpt | 45% | 35% |
+| Reserve Factor | 15% | 45% |
```
Comment (brotherlymite) [inline src/20240426_AaveV3Ethereum_UpdatingWeETHRiskParameters/UpdatingWeETHRiskParameters.md:28]: @Rozengarden the table formatting seems to be broken

File: src/20240426_AaveV3Ethereum_UpdatingWeETHRiskParameters/UpdatingWeETHRiskParameters.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Expanding borrow capacity will result in a higher collateral yield, which in turn creates a positive flywheel effect on supply and demand. By maintaining the borrow cap at an optimal ratio (35%) of the supply cap, we can balance risk, available liquidity, and LP yield effectively.
+
+We also propose to increase weETH RF to 45% to contribute to Aave DAO revenue.
+
+## Specification
+
+The following changes will be made to the **weETH** asset on Aave EthereumV3:
+
+| Parameters | Current | Recommended |
```
Comment (brotherlymite) [inline src/20240426_AaveV3Ethereum_UpdatingWeETHRiskParameters/UpdatingWeETHRiskParameters.md:24]:
```suggestion
| Parameters | Current | Recommended |
| -------------- | ------- | ----------- |
```

File: src/20240426_AaveV3Ethereum_UpdatingWeETHRiskParameters/UpdatingWeETHRiskParameters.md (hunk 1 above)
Comment (Rozengarden) [reply src/20240426_AaveV3Ethereum_UpdatingWeETHRiskParameters/UpdatingWeETHRiskParameters.md]: it was indeed missing, the "| --- | --- |..." line

**Review by ianflexa — DISMISSED** [review]

File: Makefile [hunk 3]
```diff
@@ -11,8 +11,8 @@ test   :; forge test -vvv
 test-contract :; forge test --match-contract ${filter} -vvvv
 
 # Deploy
-deploy-ledger :; forge script ${contract} --rpc-url ${chain} $(if ${dry},--sender 0x25F2226B597E8F9514B3F68F00f494cF4f286491 -vvvv,--broadcast --ledger --mnemonic-indexes ${MNEMONIC_INDEX} --sender ${LEDGER_SENDER} --verify -vvvv --slow)
-deploy-pk :; forge script ${contract} --rpc-url ${chain} $(if ${dry},--sender 0x25F2226B597E8F9514B3F68F00f494cF4f286491 -vvvv,--broadcast --private-key ${PRIVATE_KEY} --verify -vvvv --slow)
+deploy-ledger :; forge script ${contract} --rpc-url ${chain} $(if ${dry},--sender 0x25F2226B597E8F9514B3F68F00f494cF4f286491 -vvvv, --ledger --mnemonic-indexes ${MNEMONIC_INDEX} --sender ${LEDGER_SENDER} --verify -vvvv --slow --broadcast)
+deploy-pk :; forge script ${contract} --rpc-url ${chain} $(if ${dry},--sender 0x25F2226B597E8F9514B3F68F00f494cF4f286491 -vvvv, --private-key ${PRIVATE_KEY} --verify -vvvv --slow --broadcas)
```
Comment (brotherlymite) [inline Makefile:15]:
```suggestion
deploy-pk :; forge script ${contract} --rpc-url ${chain} $(if ${dry},--sender 0x25F2226B597E8F9514B3F68F00f494cF4f286491 -vvvv, --private-key ${PRIVATE_KEY} --verify -vvvv --slow --broadcast)
```

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**